etcd/build.gradle

```diff
@@ -2,9 +2,9 @@ dependencies {
     annotationProcessor "io.micronaut:micronaut-inject-java"
     annotationProcessor "io.micronaut.docs:micronaut-docs-asciidoc-config-props:$micronautDocsVersion"
 
-    api "io.micronaut:micronaut-inject"
-    api "io.micronaut:micronaut-inject-java"
-    compile "io.etcd:jetcd-core:$jetcdVersion"
+    implementation "io.micronaut:micronaut-inject"
+    implementation "io.micronaut:micronaut-inject-java"
+    implementation "io.etcd:jetcd-core:$jetcdVersion"
```

Comment:
Be careful with this change. In general Micronaut dependencies should be `api` so you don't leak the different versions to applications using this library. 
You want to keep etcd as `implementation`, as you did, because you want it to be on the applications compile classpath.

For example, in micronaut-mqtt, the eclipse-path library is `implementation` because we want it to be transitively exposed and available in users' applications: https://github.com/micronaut-projects/micronaut-mqtt/blob/master/mqttv3/build.gradle#L6
But Micronaut dependencies are on `api` scope: https://github.com/micronaut-projects/micronaut-mqtt/blob/master/mqtt-core/build.gradle#L5

Reply:
I fact, I had to do it at least for `micronaut-inject-java` because it was failing at the time to use the JAR on my demo Java app. I'll tune it a bit more.

Thanks as usual!